### DISCUSSION TITLE
mario ksrt 64 usa not loading even when its a z64 file

### DISCUSSION THREAD

**Discussion by libi1274646636363** [mupen64plus-ae/mupen64plus-ae#980] [Q&A]
I just downloaded my mario kart 64 usa in my android and when entered my M64Plus FZ i scanned it and its not adding my mario kart 64 usa version into the emulator dont know why

**Comment by fzurita** [comment]
Ok, first of all, are you using the latest version of the app here? https://play.google.com/store/apps/details?id=org.mupen64plusae.v3.fzurita

Next, what phone do you have and what Android version?

Next, do you have a valid ROM file/? What is the file size of the ROM?
